cli/dts/lib.deno.ns.d.ts

```diff
@@ -113,26 +113,47 @@ declare namespace Deno {
    * See: https://no-color.org/ */
   export const noColor: boolean;
 
-  export interface TestDefinition {
-    fn: () => void | Promise<void>;
+  export interface Tester {
```

Comment:
We need a different name, Tester is the name of the actual main test runner type.

Reply:
Do you have a different suggestion? `TestContext`?

Reply:
`TestContext` or just `Test`

Reply:
Renamed to `TestContext`.